packages/storage/amplify_storage_s3/example/integration_test/download_data_test.dart

```diff
@@ -132,6 +132,23 @@ void main() {
           expect(utf8.decode(downloadResult.bytes), 'data');
           expect(downloadResult.downloadedItem.path, publicPath);
         });
+
+        testWidgets('multi bucket', (_) async {
+          final mainBucket =
+              StorageBucket.fromOutputs('Storage Integ Test main bucket');
+
+          final downloadResult = await Amplify.Storage.downloadData(
+            path: StoragePath.fromIdentityId(
+              (identityId) => 'private/$identityId/$identityName',
+            ),
+            options: StorageDownloadDataOptions(bucket: mainBucket),
```

Comment:
Q: should this be tested on the secondary bucket instead? otherwise this test targets the same bucket as the previous ones since they default to the main bucket with no bucket specified

Reply:
Good question, let me include a to-do to add the secondary bucket after upload file/data is merged. Right now there's no way to upload to the secondary bucket.